test_scripts/RC/OnRemoteControlSettings/021_Release_resource_on_exit_app.lua

```diff
@@ -0,0 +1,59 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/10
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/resource_allocation.md
+-- Item: Use Case 3: Alternative Flow 1
+--
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description:
+-- In case:
+-- SDL received OnExitApplication (either user exits RC_app_1 vis HMI or due to driver distraction violation)
+--
+-- SDL must:
+-- 1) SDL assigns HMILevel NONE to RC_app_1 and releases module_1 from RC_app_1 control
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local exitAppReasons = {"USER_EXIT", "DRIVER_DISTRACTION_VIOLATION"}
+
+--[[ Local Functions ]]
+local function ptu_update_func(tbl)
+  tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID] = commonRC.getRCAppConfig()
+end
+
+local function exitApp(pReason, pAppId, self)
```

Comment:
@dboltovskyi  
1. Parameter `pAppId` was added as second parameter for exitApp function to be convenient with other common RC functions (all functions in commonRC module pass pAppId as second parameter).  

2. Current function is used only in one test script for now.  I propose move it into commonRC module in case this function will be used second time.

A would to leave these as is, if you do not insist.